lib/Document.php

```diff
@@ -2,15 +2,25 @@
 
 namespace Sjuper;
 
-class Document 
+class Document implements Connectable
 {
 
     /** @var string $title */
     private $title;
 
-    public function __construct($title)
+    /** @var string $title */
+    private $id;
+
+    public function __construct($title, $id)
     {
         $this->title = $title;
+        $this->id = $id;
+    }
+
+    public function id()
+    {
```

Comment:
:no_entry_sign: **Error**: Tabs must be used to indent lines; spaces are not allowed (*Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed*).